### DISCUSSION TITLE
How to remove NvimTree_1 from jumplist?

### DISCUSSION THREAD

**Discussion by jugarpeupv** [nvim-tree/nvim-tree.lua#3032] [Q&A]
Ey, does anyone know how to remove NvimTree_1 from jumplist?

When i enter neovim and run Jumplist:

<img width="1463" alt="image" src="https://github.com/user-attachments/assets/98ea7349-139b-4e9f-9e9f-6afcd93dd273" />

**Comment by jugarpeupv** [comment]
I thought i had it by setting bufhidden = delete but no, it is not working

**Comment by alex-courtis** [comment]
`:Telescope buffers` does not show nvim-tree or other similar windows.

Perhaps there is some common configuration in Jumplist to achieve the same buffer list.

**Reply by jugarpeupv** [reply]
@alex-courtis i dont see any option for that

I tried with <cmd>keepjumps NvimTreeToggle<cr> but it does not work

Maybe keepjumps should be implemented inside the nvimtree code?

It is annoying because if you enter nvim, you see nvim tree, you open a file, then ctrl+o and then you end up with this

<img width="1288" alt="image" src="https://github.com/user-attachments/assets/22324453-1e80-4472-bac0-67f36ce61686" />

**Reply by alex-courtis** [reply]
> Maybe keepjumps should be implemented inside the nvimtree code?

That sounds interesting and would likely resolve your issue.

I'd be grateful for a proof of concept; please see [CONTRIBUTING.md](https://github.com/nvim-tree/nvim-tree.lua/blob/master/CONTRIBUTING.md)
